core/src/main/java/org/apache/calcite/util/RelToSqlConverterUtil.java

```diff
@@ -68,6 +68,41 @@ public static void unparseHiveTrim(
     }
   }
 
+  /**
+   * For usage of TRIM, LTRIM and RTRIM in BQ see
+   * <a href="https://cloud.google.com/bigquery/docs/reference/standard-sql/functions-and-operators#trim">
+   * BQ Trim Function</a>.
+   */
+  public static void unparseTrim(SqlWriter writer, SqlCall call, int leftPrec,
```

Comment:
Because you moved the method you need to revisit its javadoc.

Reply:
Here had changed in other PR: https://github.com/apache/calcite/pull/4360  would rebase it after it merge.